src/polytopes/ngon.jl

```diff
@@ -80,6 +80,33 @@ function measure(t::Triangle{3})
   norm((B - A) × (C - A)) / 2
 end
 
+function circumcenter(t::Triangle{2})
+  A, B, C = t.vertices
+  sin2A = sin(2 * ∠(C, A, B))
+  sin2B = sin(2 * ∠(A, B, C))
+  sin2C = sin(2 * ∠(B, C, A))
+  sines = [sin2A, sin2B, sin2C]
```

Comment:
```suggestion
  sines = sin(2 .* innerangles(boundary(t)))
```
Maybe this is more straightforward? Its worth checking if the performance is similar.